src/main/java/dev/slimevr/vr/processor/skeleton/LegTweakBuffer.java

```diff
@@ -654,4 +700,135 @@ private float getRightFootLockLiklyHood() {
 			/ (MAX_SCALAR_ACTIVE - MIN_SCALAR_ACTIVE)
 			- PARAM_SCALAR_MID;
 	}
+
+	// get the pressure prediction for the feet based of the center of mass
+	// (assume mass is 1)
+	// for understanding in the future this is assuming that the mass is one and
+	// the force of gravity
+	// is 9.8 m/s^2 this allows for the force sum to map directly to the
+	// acceleration of the center of mass
+	// since F = ma and if m is 1 then F = a
+	private float[] getPressurePrediction() {
+		float leftFootPressure = 0;
+		float rightFootPressure = 0;
+
+		// get the vector's from the com to each foot
+		Vector3f leftFootVector = leftFootPosition.subtract(centerOfMass).normalize();
+		Vector3f rightFootVector = rightFootPosition.subtract(centerOfMass).normalize();
+
+		// get the magnitude of the force on each foot
+		float leftFootMagnitude = 9.81f * leftFootVector.y / leftFootVector.length();
+		float rightFootMagnitude = 9.81f * rightFootVector.y / rightFootVector.length();
```

Comment:
Could this use the `gravity` constant? Either that or extracting `9.81f` to a constant of its own would be nice